dbt_platform_helper/providers/platform_config_schema.py

```diff
@@ -434,6 +436,22 @@ def valid_s3_bucket_name(name: str):
 
         return True
 
+    @staticmethod
+    def __datadog_schema() -> dict:
+        return {
+            "type": "datadog",
+            Optional("environments"): {
+                PlatformConfigSchema.__valid_environment_name(): {
+                    Optional("team_name"): str,
+                    Optional("contact_name"): str,
+                    Optional("contact_email"): str,
+                    Optional("repository"): str,
+                    Optional("docs"): str,
```

Comment:
Maybe...
```suggestion
                    Optional("codebase_documentation_url"): str,
```

Reply:
Every repository should have a README, even if it links out to documentation elsewhere.

Would is be fair to suggest that we don't need the docs option here?

Reply:
This field is more for generic documentation about the application.  Maybe pointing to a confluence page or a run book.